transmart-core-db/grails-app/services/org/transmartproject/db/RestExportService.groovy

```diff
@@ -0,0 +1,117 @@
+package org.transmartproject.db
+
+import grails.transaction.Transactional
+import groovy.json.JsonException
+import groovy.json.JsonSlurper
+import org.springframework.beans.factory.annotation.Autowired
+import org.transmartproject.core.dataquery.highdim.HighDimensionResource
+import org.transmartproject.core.dataquery.highdim.assayconstraints.AssayConstraint
+import org.transmartproject.core.exceptions.InvalidArgumentsException
+import org.transmartproject.core.exceptions.NoSuchResourceException
+import org.transmartproject.core.ontology.OntologyTerm
+import org.transmartproject.core.ontology.OntologyTermsResource
+import org.transmartproject.export.Datatypes
+import org.transmartproject.export.Tasks.DataExportFetchTask
+import org.transmartproject.export.Tasks.DataExportFetchTaskFactory
+
+import static org.transmartproject.core.ontology.OntologyTerm.VisualAttributes.HIGH_DIMENSIONAL
+
+@Transactional
+class RestExportService {
+
+    @Autowired
+    DataExportFetchTaskFactory dataExportFetchTaskFactory
+
+    @Autowired
+    HighDimensionResource highDimensionResourceService
+
+    OntologyTermsResource ontologyTermsResource
+
+    List<File> export(arguments) {
```

Comment:
Type of arguments?

Reply:
I've removed service altogether as it's not used.